save-demo/src/main/kotlin/com/saveourtool/save/demo/utils/KubernetesUtils.kt

```diff
@@ -129,10 +127,23 @@ private fun demoAgentContainerSpec(
     image = imageName
     imagePullPolicy = "IfNotPresent"
 
-    val startupCommand = downloadAndRunAgentCommand(agentDownloadUrl, AgentType.DEMO_AGENT)
+    val envOptions = sequenceOf(
+        "KTOR_LOG_LEVEL" to "TRACE",
+    )
+
+    val startupCommand = downloadAndRunAgentCommand(agentDownloadUrl, AgentType.DEMO_AGENT, envOptions = envOptions)
 
     command = listOf("sh", "-c", startupCommand)
 
+    ports = listOf(
+        ContainerPort().apply {
```

Comment:
maybe create some extension like `ContainerPort().default()` or smth similar with these values
could we?

Reply:
Makes sense, will do it

Reply:
Done